airbyte-integrations/connectors/source-mssql/src/test/java/io/airbyte/integrations/source/mssql/MssqlDataSourceFactoryTest.java

```diff
@@ -25,7 +28,8 @@ protected void testCreatingDataSourceWithConnectionTimeoutSetBelowDefault() {
         container.getPassword(),
         container.getDriverClassName(),
         container.getJdbcUrl(),
-        connectionProperties);
+        connectionProperties,
+        CONNECTION_TIMEOUT);
```

Comment:
I'm surprised the test passes with this change. I expect you'd need to pass new `MsSqlSource.getConnectionTimeout(connectionProperties)`